src/docs/source/includes/_deployment.md

```diff
@@ -262,8 +262,6 @@ secret to the `mushop-utilities` namespace:
       --set global.osb.compartmentId=<compartmentId>
     ```
 
-    > Note that the `oci-credentials` secret was created [previously](#provisioning)
```

Comment:
We need a note saying where the oci-credentials is coming from.

Reply:
It is there in step 1 for OSB